node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeadmin/NodeAdminImpl.java

```diff
@@ -235,7 +235,11 @@ void synchronizeNodeSpecsToNodeAgents(
                 existingContainers.stream(), containerName -> containerName);
 
         final Set<ContainerName> obsoleteAgentContainerNames = diff(nodeAgents.keySet(), new HashSet<>(hostnameByContainerName.keySet()));
-        obsoleteAgentContainerNames.forEach(containerName -> nodeAgents.remove(containerName).stop());
```

Comment:
This seems redundant if there is also log in `stop()` ?

Reply:
Absolutely, fixed